lib/svg-preview-view.js

```diff
@@ -64,10 +75,24 @@ class SvgPreviewView extends View {
     this.disposables.add(atom.tooltips.add(this.whiteButton[0], { title: "Use white transparent background" }))
     this.disposables.add(atom.tooltips.add(this.blackButton[0], { title: "Use black transparent background" }))
     this.disposables.add(atom.tooltips.add(this.transparentButton[0], { title: "Use transparent background" }))
+    this.disposables.add(atom.tooltips.add(this.zoomInButton[0], { title: "Zoom in" }))
+    this.disposables.add(atom.tooltips.add(this.zoomOutButton[0], { title: "Zoom out" }))
+    this.disposables.add(atom.tooltips.add(this.zoomResetButton[0], { title: "Reset zoom level" }))
```

Comment:
I don't think tooltips for the zoom buttons are necessary. The original ImageView does have theme either.